domains/pearlcz.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "0pearlcz0",
+           "email": "",
+           "discord": "809488731353776149",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.UKtO0TGQ2vw45um_Y1n0Pj66OTCqJlpmyrIFY2TU8nzaMXYf40P4I0jdTeI0begjgJSK98WBExHDHTKQEAljN1LZHpDF0nZQD0xg7uRnqemy3lGceshf0QIs-Tyq5ipavx9CjosXoEMwtiqhmFDKS57sLtI37wtT0f-zambKmBiRFR18gzxV3AuBgpLSASoR-PWbJfMf64BCekXdUrrZ352BxRp9tjrAkxxK7iMcSNJn69mKkMdLlLu92X67WGU5MTKctUfrFOZS_pE1UZyAGzZodmCiS3QiKJMo-ZXvZyy7VzSB4EYgYcLYX1jjsgTmoPl-EXnyk1KAYcVDvEJziQ.oI8DhkP6hv1aeLuXl95Q5w.stm_-qvuO3CsAT1ZZwfjgJdrXx6fbwc8M6ORnJQE6MB6dAWTYyl0bYvBUJUs5nrC4sUl2wg5xoV68td2d4z4CJYYf-sUV_cXyf2gRmCGV94.TGY8u9yodBgKz4fGvldIfQ"
+        },
+    
+        "record": {
+            "CNAME": "ns1.byet.org"
```

Comment:
Wrong way to use NS records.

Reply:
I want this:
Register _github-pages-challenge-0pearlcz0.pearlcz.is-a.dev with TXT record pointing to bf6f41543a58bffd08fd85ffd9d254.

Reply:
There is a second Commit.

Reply:
You have 2 files, one of them are incorrect. We don't merge incorrect files.

Reply:
The first is outdated but i don't know how to change it or delete it.

Reply:
Theres always going to your fork, going to `domains` directory, finding the incorrect file and updating that y'know?
Also I won't merge the TXT verification one if you delete the main file instead of fixing it since theoretically it doesn't exist, resulting in that you cannout make one.

Reply:
So what i need to make my domain with TXT?

Reply:
It should be all done. Sorry for the problems.